lang/en_us/format_topcoll.php

```diff
@@ -67,3 +67,6 @@
 // Capabilities.
 $string['topcoll:changecolour'] = 'Change or reset the color';
 
+// Toggle Display Blocks
+$string['defaultdisplayblocks'] = 'Display the four blocks that come with this format plugin';
```

Comment:
Not needed as no change from en version and thus Moodle uses that instead if no entry here.